Dockerfile

```diff
@@ -1,17 +1,17 @@
-FROM debian:trixie-slim as builder
+FROM debian:trixie-slim AS builder
 LABEL maintainer=nils@gis-ops.com
 
 WORKDIR /
 
 RUN echo "Updating apt-get and installing dependencies..." && \
-  apt-get -y update > /dev/null && apt-get -y install > /dev/null \
-  git-core \
-  build-essential \
-	g++ \
-  libssl-dev \
-	libasio-dev \
-  libglpk-dev \
-	pkg-config
+    apt-get -y update > /dev/null && apt-get -y install > /dev/null \
+    git-core \
+    build-essential \
+    g++ \
+    libssl-dev \
+    libasio-dev \
+    libglpk-dev \
+    pkg-config
```

Comment:
This looks like an unrelated indentation change.

Reply:
Yes, this not related to the warning.
I took to liberty to unify the mix of tab and spaces, and 2 space / 4 space indentations. It did not look like it was done intentionally.

Reply:
I did not notice the tabs, looks like it's more consistent now indeed.